velox/dwio/common/compression/CMakeLists.txt

```diff
@@ -16,4 +16,4 @@ add_library(velox_dwio_common_compression Compression.cpp PagedInputStream.cpp
                                           PagedOutputStream.cpp)
 
 target_link_libraries(velox_dwio_common_compression velox_dwio_common xsimd
-                      gtest Folly::folly)
+                      Folly::folly)
```

Comment:
Looks good, was it something that was added inadvertantly ?

Reply:
Yes, likely an oversight. This is the PR that introduced this file https://github.com/facebookincubator/velox/pull/5946/files#diff-c4931d0d8ed2030f48435f4c191b1ca8d8d0b0dbd39a7868f27383e7ace157ef
I have seen this issue previously. I was able to track this today.